openedx.yaml

```diff
@@ -1,4 +1,7 @@
+# openedx.yaml
 
+---
+nick: openedx-calc
```

Comment:
OEP-2 says this about the `nick` field:

> Should be 3-4 characters, because this value is used in charts with very limited space on their axes.

Reply:
Oh.  I can remove this.  I took this from the cookie-cutter, which probably needs to be fixed then: https://github.com/edx/cookiecutter-django-app/blob/master/%7B%7Bcookiecutter.repo_name%7D%7D/openedx.yaml#L5

Reply:
@jmbowman: I removed this change (see my comment above).  Is this ready for a thumb, or are there other issues?